open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-api/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/ffdc/AnalyticsModelingErrorCode.java

```diff
@@ -116,16 +116,61 @@
 			"Review the exception to resolve the issue."),
 	FAILED_FETCH_DATABASE_SCHEMAS(
 			"OMAS-ANALYTICS-MODELING-017", 
-			"Schemas for database {} could not be fetched.",
+			"Schemas for database {0} could not be fetched.",
 			"The system is unable to process the request.", 
 			"Review the exception to resolve the issue."),
-
+	FAILED_FIND_DATABASE_SCHEMA(
```

Comment:
These error codes need more explanation to explain what he system was doing and how a user resolved the issue.  The text in the explanations should be different for each message.